### DISCUSSION TITLE
No More Updates?

### DISCUSSION THREAD

**Discussion by LutraVibes** [rafaelvcaetano/melonDS-android#1284] [Q&A]
Hello, Rafael!

I've been noticing that updates haven't been coming around recently. Are you okay? Did you quit this project?

Best wishes,
Lutra

**Comment by christopher3242** [comment]
Not yet . Just may updates.

**Comment by LutraVibes** [comment]
Just "may" updates?

**Reply by christopher3242** [reply]
all right @WinUser-Generic
